pepper-apis/docs/specification/src/endpoints/user.studies.activity.instance.question-options.yml

```diff
@@ -0,0 +1,36 @@
+get:
+  operationId: getOptionsForActivityInstanceQuestion
+  tags:
+    - Suggestion
+  summary: get a list of question options from server to select
+  description: For specific Activity Instance Question - returns a list of options that can be selected as an answer to this specific question
+  parameters:
+    - in: path
+      name: user
+      required: true
+      description: the user's unique identifier
+      schema:
+        type: string
+    - in: path
+      name: study
+      required: true
+      description: the unique identifier for the study
+      schema:
+        type: string
+    - in: path
+      name: instance
+      required: true
+      description: the unique identifier for the instance
+      schema:
+        type: string
+    - in: path
+      name: stableId
+      required: true
+      description: the activity instance select question's unique identifier
+      schema:
+        type: string
+  responses:
+    200:
+      $ref: '../pepper.yml#/components/responses/ActivityInstanceSelectAnswersResponse'
```

Comment:
Is this the correct response type for the 200 code?

Reply:
Not sure. I think we could re use this class `ActivityInstanceSelectAnswerSubmission` and just change the file's name for a more appropiate one:

![image](https://user-images.githubusercontent.com/98110251/152220896-d1033d03-1dd1-4e16-ac7d-56ef7db1b245.png)

What do you think?

Reply:
`ActivityInstanceSelectAnswerSubmission` is not an appropriate template for this situation. First off, the `ActivityInstanceSelectAnswerSubmission` response is used when responding to the requests PATCHed to the `/answers` endpoint- we aren't doing that here. Further, at a minimum, the response from the `/options` endpoint should be returning a list of `PicklistOption` objects which match the request's `q` query parameter. Given that the `PicklistOption`s may also be part of a group, those groups should be included in the response as well.

Reply:
I'd suggest checking out the `getDrugSuggestions` and `getCancerSuggestions` endpoints for the general pattern we've used in the past.

Reply:
@bskinner Is it possible to do this :  `List<PicklistOption, PicklistGroup> results` in java?

![image](https://user-images.githubusercontent.com/98110251/152417655-515f58c6-9ceb-4ded-ba5e-9d24a829acd4.png)

Reply:
No- the [java.util.List](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/List.html) class only supports a single type as a generic parameter. Structures like tuples are not supported in Java.